src/selectors/MovieSelectors.js

```diff
@@ -0,0 +1,3 @@
+export const getFav = (state) => {
```

Comment:
Same thing about abbreviating.
Renaming it to `getFavoriteMovies` is my suggestion: seems more precise to me.

Reply:
Done!